Jenkinsfile

```diff
@@ -121,13 +121,15 @@ node {
             --create-artifacts
       """
 
-      // Run the e2e test framework
-      sh """${env.ROOT}/contrib/jenkins/run_e2e.sh \
-            --registry gcr.io/${test_project}/catalog/ \
-	    --version ${version} \
-	    --cleanup \
-	    --create-artifacts
-      """
+      ansiColor('xterm-darker-gray') {
```

Comment:
Is this going to mean that the color codes still show up in the log?  I'd love a colorized jenkins log if I could get it, but I don't think I've ever seen a jenkins do that.